rcl/test/rcl_add_custom_gtest.cmake

```diff
@@ -67,9 +67,14 @@ macro(rcl_add_custom_gtest target)
   if(_ARG_APPEND_LIBRARY_DIRS)
     set(_ARG_APPEND_LIBRARY_DIRS "APPEND_LIBRARY_DIRS" ${_ARG_APPEND_LIBRARY_DIRS})
   endif()
+  if(_ARG_SKIP_TEST)
+    set(_ARG_SKIP_TEST "SKIP_TEST")
+  else()
+    set(_ARG_SKIP_TEST "")
```

Comment:
for some reason before adding this, `"FALSE"` was passed to the script instead of an empty string, resulting in the following error: `run_test.py: error: --env argument 'FALSE' contains no equal sign` 
haven't found why FALSE is passed yet